### DISCUSSION TITLE
About stopped services in extra settings

### DISCUSSION THREAD

**Discussion by zAlfok** [seerge/g-helper#3346] [Q&A] [answered]
Hi, when using the stop running services button it seem to work fine. However, when looking at the task manager there is one service is still running: AsusPTPService. 
I go from this:
![image](https://github.com/user-attachments/assets/267cc3f3-bf04-4837-8b9f-694029aef69e)
To this
![image](https://github.com/user-attachments/assets/87d6e651-3eca-460d-8b3c-d57c23a2a916)
I'm using Zephyrus G14 - G401II and the online ASUS related "software" that I has installed is RefreshRateService (talking about programs listed in Windows Add or remove programs)
![image](https://github.com/user-attachments/assets/ffd9910c-f97d-445f-b51b-316c7a2aa649)
So this makes me wonder a couple of things:
First, what does this service do? Does anyone have this service and have it stopped? Is there any risk in stopping it? I tried to look for the description in the services, but unlike the others, it doesn't have one
![image](https://github.com/user-attachments/assets/e595e7b4-578b-4aed-af85-3b85eaee486c)
Second, what services are listed here? [https://github.com/seerge/g-helper/wiki/FAQ#what-services-can-be-stopped-from-the-extra-settings](url) Just the main/common ones found in most ASUS devices? Just MyAsus related? Or is a bug and that service should have been stopped too? 
Finally, maybe not so related, is RefreshRateService actually a service? What is it suppossed to do? Can it be uninstalled?

**Comment by seerge** [comment, marked as answer]
@zAlfok hello, 

G-Helper can stop only services that will be running on your system anyway (installed with Asus System Control Interface). It doesn't stop / uninstall neither Armoury nor it's components. 

I don't know what does AsusPTPService does :) But names suggests it's something about peer to peer ? whatever it means. If you don't need it - you can try to stop it and see what happens.

RefreshRateService seem like an older version of Asus Smart Display Control app, that changes refresh rate to 60hz on battery, and back to maximum when plugged in. That one is not needed for G-Helper, as it can change refresh rate by itself using windows API.

**Reply by zAlfok** [reply]
Thank you very much @seerge, it's much clearer now :)
